### DISCUSSION TITLE
Problems with Awakend Trade Macro

### DISCUSSION THREAD

**Discussion by Neox91295** [SnosMe/awakened-poe-trade#711] [Q&A] [answered]
Hello Guys,

i tried the Awakend Trade Macro. I got the Problem that all Hotkeys working except the CRTL+D command. And I dont have any possibility to go to the sttings, becouse i dont have any thing on my screen after i press the comand Shift+Space

I woud apreciate any help, so i can use the Macro. 

If more Information is need just ask me.

Greets

**Comment by SnosMe** [comment]
https://snosme.github.io/awakened-poe-trade/issues

**Reply by Neox91295** [reply]
I already tried this all. I cant even get to settings couz i dont have the cog icon. And all other macros doing good. It is only the CRTL+D command. I have no Log and and can copy the hole item with CRTL+C.

**Reply by SnosMe** [reply]
No log file, probably this, then https://github.com/SnosMe/awakened-poe-trade/issues/594#issuecomment-1122275974

**Comment by Neox91295** [comment]
At my PC i have 4 process working and all other command (F5 / F9) working it is literly only the CTRL+D command what isnt working. I atachts phots of the process and then Shift+space window aswell as the window what shows if i try to get to setting. Hopefuly we get to get this working

![Settings](https://user-images.github
![Awakend Process](https://user-images.githubusercontent.com/111895235/186252106-25166411-06c2-424a-ac3c-3e2fb11bd230.jpg)
usercontent.com/111895235/186252093-1adb6a30-37bf-49e4-9a39-ffa21f9f3850.jpg)
![Shift and Space window](https://user-images.githubusercontent.com/111895235/186252098-a0f740b2-6a3b-46d4-b307-b74234064015.jpg)
s

**Comment by Neox91295** [comment]
![Settings](https://user-images.githubusercontent.com/111895235/186252235-2a2beccf-9cbd-4458-94bd-fb0bc7afa16a.jpg)

**Comment by Neox91295** [comment]
![Log1](https://user-images.githubusercontent.com/111895235/186252726-443f6d88-145a-45a7-990a-5d305fc6f09c.jpg)

![Log2](https://user-images.githubusercontent.com/111895235/186252745-58cf44be-9e5a-463b-b5d8-fec719855d24.jpg)

**Comment by SnosMe** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/4292308/186267082-b26f0136-bda6-4b3f-b3e6-9b811fb99348.png)

Yeah, no cog icon here is suspicious.
Try removing 

![image](https://user-images.githubusercontent.com/4292308/186267310-1f042b7e-ecb3-4fb2-9477-5d67b7871b67.png)

and restart

**Reply by Neox91295** [reply]
Now it working finaly. 

Thanks for ur assist.
